lib/internal/Magento/Framework/Logger/Test/Unit/Handler/SystemTest.php

```diff
@@ -27,20 +29,22 @@ class SystemTest extends TestCase
     private $filesystemMock;
 
     /**
-     * @var Exception|Mock
+     * @var ExceptionHandler|Mock
      */
     private $exceptionHandlerMock;
 
     /**
      * @inheritdoc
+     *
+     * @throws Exception
      */
     protected function setUp(): void
     {
         $this->filesystemMock = $this->getMockBuilder(DriverInterface::class)
             ->getMockForAbstractClass();
-        $this->exceptionHandlerMock = $this->getMockBuilder(Exception::class)
-            ->disableOriginalConstructor()
-            ->getMock();
+        $this->exceptionHandlerMock = $this->getMockBuilder(
+            ExceptionHandler::class
+        )->disableOriginalConstructor()->getMock();
```

Comment:
Can we replace it with the following?
```suggestion
        $this->exceptionHandlerMock = $this->createMock(ExceptionHandler::class);
```